### PR TITLE
docstring changes in accelerators

### PR DESCRIPTION
**Pull request by AlKun25**
## What does this PR do?

Part of #5434 

- [x] \_\_init\_\_.py
- [x]  accelerator.py
- [x] cpu.py
- [x] gpu.py
- [x] tpu.py

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by AlKun25** [comment]
I am not understanding why so many tests are failing, I only made a slight change in code cause PEP8 test suggested me to do so, rest all is just documentation 
Would like to know if these are failing due to my changes?

**Review by tchaton — APPROVED** [review]
LGTM !

**Comment by tchaton** [comment]
> I am not understanding why so many tests are failing, I only made a slight change in code cause PEP8 test suggested me to do so, rest all is just documentation
> Would like to know if these are failing due to my changes?

Dear @AlKun25,

Your changes are not responsible for the failing CI.
An update in TorchText + MNIST Download HTTPError are causing those issues.


The changes you seem to be fine ! Thanks for your contributions !!!
Screenshot 2021-03-03 at 17 51 25

Best,
T.C

**Review by SeanNaren — APPROVED** [review]
Thanks!!

**Review by akihironitta — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

File: pytorch_lightning/accelerators/cpu.py [hunk 1]
```diff
@@ -12,6 +12,12 @@
 class CPUAccelerator(Accelerator):
 
     def setup(self, trainer: 'Trainer', model: 'LightningModule') -> None:
+        """
+        Raises:
+            MisconfigurationException:
+                If amp is used with cpu, or
+                if the selected device is not CPU.
```
Comment (Borda) [inline pytorch_lightning/accelerators/cpu.py:19]:
```suggestion
                If AMP is used with CPU, or if the selected device is not CPU.
```

File: pytorch_lightning/accelerators/tpu.py [hunk 2]
```diff
@@ -21,6 +21,12 @@
 class TPUAccelerator(Accelerator):
 
     def setup(self, trainer: 'Trainer', model: 'LightningModule') -> None:
+        """
+        Raises:
+            MisconfigurationException:
+                If amp is used with tpu, or
+                if TPUs are not using single TPU core or TPU spawn training.
```
Comment (Borda) [inline pytorch_lightning/accelerators/tpu.py:28]:
```suggestion
                If AMP is used with TPU, or if TPUs are not using a single TPU core or TPU spawn training.
```

**Review by Borda — APPROVED** [review]
